ex20.py

```diff
@@ -0,0 +1,33 @@
+from sys import argv
+
+script, input_file = argv
+
+def print_all(f):
+	print f.read()
+
+# seek(n) to read a file's content from byte-n
+def rewind(f):
+	f.seek(0)
+
+def print_a_line(line_count, f):
+	print line_count, f.readline()
+
+current_file = open(input_file)
```

Comment:
I am also a newbie. I just learned a good way for this. It's with ... as ...: 
with open( input_file) as current_file: